src/kibana/components/vislib/visualizations/_point_series_chart.js

```diff
@@ -15,6 +15,30 @@ define(function (require) {
       PointSeriesChart.Super.apply(this, arguments);
     }
 
+    PointSeriesChart.prototype._stackMixedValues = function (stackCount) {
+      var currentStackOffsets = [0, 0];
+      var currentStackIndex = 0;
+
+      return function (d, y0, y) {
+        var firstStack = currentStackIndex % stackCount === 0;
+        var lastStack = ++currentStackIndex === stackCount;
+
+        if (firstStack) {
+          currentStackOffsets = [0, 0];
+        }
+
+        if (lastStack) currentStackIndex = 0;
```

Comment:
@spalger I haven't been able to find a way that this works correctly without using `lastStack`.